src/mtx/commands/uplink/getMeterInfo.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Uplink command to get meter information.
+ *
+ * The corresponding downlink command: `getMeterInfo`.
+ *
+ * @packageDocumentation
+ *
+ * @example create command instance from command body hex dump
+ * ```js
+ * import * as getMeterInfo from 'jooby-codec/mtx/commands/uplink/getMeterInfo.js';
+ *
+ * // simple response
+ * const bytes = [0x00];
+ *
+ * // decoded payload
+ * const parameters = getMeterInfo.fromBytes(bytes);
+ *
+ * console.log(parameters);
+ * // output:
+ * {ten: 0}
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/GetMeterInfo.md#response)
+ */
+
+import * as command from '../../utils/command.js';
+import * as types from '../../types.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+
+
+interface IGetMeterInfo {
+    /** Integration period for load profiles */
```

Comment:
```suggestion
    /** Integration period for load profiles (also present in {@link IOperatorParameters}) */
```

and don't forget to import IOperatorParameters

Reply:
done